pkg/apis/serving/v1beta1/predictor.go

```diff
@@ -68,7 +68,6 @@ type PredictorSpec struct {
 	// Component extension defines the deployment configurations for a predictor
 	ComponentExtensionSpec `json:",inline"`
 }
-
```

Comment:
Keep this line?

Reply:
do we need?

Reply:
maybe it is good to keep as well, new line between the structs.

Reply:
Yes, a new line would be clearer

Reply:
updated